code/modules/mob/living/carbon/xenomorph/Powers.dm

```diff
@@ -564,7 +567,10 @@
 	use_plasma(10)
 
 	if(H.stat != DEAD && (!(H.status_flags & XENO_HOST) || !istype(H.buckled, /obj/structure/bed/nest)) )
-		H.apply_damage(20)
+		var/damage = 20 + 3 * upgrade
+		if(frenzy_aura > 0)
+			damage += (frenzy_aura * 2)
+		H.apply_damage(damage)
```

Comment:
This bypasses armor.

Reply:
Yeah, was going to fix it; got distracted by the SS:CM test